extensions/wrapper/wrapper-ee-api/src/main/java/de/sovity/edc/ext/wrapper/api/ee/model/StoredFile.java

```diff
@@ -32,7 +32,7 @@
 @Schema(description = "Represents a stored file in the file storage extension")
 public class StoredFile {
     @Schema(description = "Identifier of the StoredFile object",
-            example = "toBeDefined",
+            example = "a-stored-file-001",
```

Comment:
no need to fix, just a comment

personally I feel that the indefinite article makes the ID harder to understand "stored-file-001" would've been fine